src/iocore/aio/test_AIO.cc

```diff
@@ -223,11 +223,11 @@ dump_summary()
 
   auto &m = Metrics::getInstance();
 
-  Metrics::IntType *completed = m.lookup(m.lookup("proxy.process.io_uring.completed"));
-  Metrics::IntType *submitted = m.lookup(m.lookup("proxy.process.io_uring.submitted"));
+  Metrics::Counter::AtomicType *completed = m.lookup(m.lookup("proxy.process.io_uring.completed"));
+  Metrics::Counter::AtomicType *submitted = m.lookup(m.lookup("proxy.process.io_uring.submitted"));
 
-  printf("submissions: %lu\n", Metrics::read(submitted));
-  printf("completions: %lu\n", Metrics::read(completed));
+  printf("submissions: %lu\n", Metrics::Gauge::load(submitted));
```

Comment:
Really %lu quit slacking https://en.cppreference.com/w/cpp/types/integer

Reply:
Yeh, not my code.